### PR TITLE
OR-98 Updates the logic and services which should connect to GraphDB

### PR DESCRIPTION
**Pull request by tonyKunchev**
- Updated all services which should connect to the GraphDB to use the
proxy endpoint that was added to the RDF Mapper backend.
- Added few workarounds for the places, where we need the actual address
of the remote GraphDB, in particular the links of the values in Preview
mode.
- Fixed an issue where if the user sets full URL for subject, to the URL
was prepended the base IRI making it invalid.
- Fixed the stubbed endpoints in the Cypress tests.
- Removed some code that isn't required anymore.
- Bumped the version of the application from `1.0.6` to `2.0.0` which will
be the new release, when it is done.

### CODE REVIEW

File: src/app/auth/token.interceptor.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ import {LocalStorageService} from 'src/app/services/local-storage.service';
 export class TokenInterceptor implements HttpInterceptor {
   constructor(protected localStorageService: LocalStorageService) {}
 
+  // TODO: probably won't work, not sure
```
Comment (mihailradkov) [inline src/app/auth/token.interceptor.ts:15]: This interceptor becomes useless with the extraction from GDB.

File: src/app/main/mapper/cell/cell.component.ts [hunk 2]
```diff
@@ -286,6 +291,7 @@ export class CellComponent extends OnDestroyMixin implements OnInit {
   }
 
   getGraphDbResourceUrl(previewItem): string {
-    return environment.graphDbUrl + '/resource?uri=' + this.getResourceUri(previewItem);
+    const url = this.graphDbUrl.endsWith('/') ? this.graphDbUrl.slice(0, -1) : this.graphDbUrl;
```
Comment (mihailradkov) [inline src/app/main/mapper/cell/cell.component.ts:294]: Better normalize this in the service.

File: src/app/main/mapper/cell/cell.component.ts [hunk 3]
```diff
@@ -78,15 +77,19 @@ export class CellComponent extends OnDestroyMixin implements OnInit {
...
-              private dialogService: DialogService) {
+              private dialogService: DialogService,
+              private mapperService: MapperService) {
     super();
   }
 
-
   ngOnInit(): void {
+    // this still fails to assign the correct value to the variable in specific cases
+    // Tony: I don't f@^@#$@ know how to force this to work in synchronous way
```
Comment (mihailradkov) [inline src/app/main/mapper/cell/cell.component.ts:91]: :astonished:

File: src/app/auth/token.interceptor.ts (hunk 1 above)
Comment (tonyKunchev) [reply src/app/auth/token.interceptor.ts:15]: Should I remove it or disable it somehow ?

File: src/app/main/mapper/cell/cell.component.ts (hunk 2 above)
Comment (tonyKunchev) [reply src/app/main/mapper/cell/cell.component.ts:294]: Yeah, I was debating on that. Will update it.

File: src/app/main/mapper/cell/cell.component.ts (hunk 3 above)
Comment (tonyKunchev) [reply src/app/main/mapper/cell/cell.component.ts:91]: It returns some kind of `Observable` object which can be chained further, but I don't know how to unwrap the actual result... 😭

File: src/app/main/mapper/cell/cell.component.ts (hunk 2 above)
Comment (tonyKunchev) [reply src/app/main/mapper/cell/cell.component.ts]: Refactored.

File: src/app/main/mapper/cell/cell.component.ts (hunk 3 above)
Comment (tonyKunchev) [reply src/app/main/mapper/cell/cell.component.ts]: Refactored.

File: src/app/main/mapper/cell/cell.component.ts [hunk 4]
```diff
@@ -78,17 +76,14 @@ export class CellComponent extends OnDestroyMixin implements OnInit {
 
   ViewMode = ViewMode;
 
+  private graphDbUrl: string;
```
Comment (mihailradkov) [inline src/app/main/mapper/cell/cell.component.ts:79]: I don't see this being used.

File: src/app/auth/token.interceptor.ts (hunk 1 above)
Comment (mihailradkov) [reply src/app/auth/token.interceptor.ts:15]: Remove it IMO.

Comment (tonyKunchev) [reply src/app/auth/token.interceptor.ts]: Done.

File: src/app/main/mapper/cell/cell.component.ts (hunk 4 above)
Comment (tonyKunchev) [reply src/app/main/mapper/cell/cell.component.ts]: Removed.

**Review by mihailradkov — APPROVED** [review]

**Pull request merged**